### DISCUSSION TITLE
Is it possible to limit the types of wallets that can be selected in qrcodeModal?

### DISCUSSION THREAD

**Discussion by siosio34** [WalletConnect/walletconnect-monorepo#2287] [Developer Support] [answered]
I'm currently using wagmi (v 0.12.10) and the walletconnectLegacyConnector package.

Is it possible to limit the wallet type on the wallet selection screen when QRModal appears?

For example, in opensea, when using WALLETCONNECT as shown in the picture below, only MetaMask, Rainbow, and Trust Wallet appear.

![image](https://user-images.githubusercontent.com/7166022/235644166-6a4b721b-6abb-4812-9eaf-16417f1537eb.png)

I tried using mobileLinks and desktopLinks for qrcodeModalOptions, but only win QRCode is enabled on ios if possible.
![image](https://user-images.githubusercontent.com/7166022/235644206-3171a4d5-4792-4ffa-a621-dc11a865f17e.png)

**Comment by finessevanes** [comment, marked as answer]
Hey - you can do this if you're using [Web3Modal v2](https://docs.walletconnect.com/2.0/web3modal/about)

https://docs.walletconnect.com/2.0/web3modal/options#explorerrecommendedwalletids-optional
